bin/modules/fetchToken.js

```diff
@@ -25,27 +25,52 @@ module.exports = (rememberedToken) => {
       if (!exists) {
         rej(err("No token found!"));
       } else {
+        const UNLOCK_EXISTING_TOKEN = "Unlock existing token";
```

Comment:
Sure, I can change that. The JavaScript convention for constants is all caps, spaced by underscores, but I'm happy to deviate from that if you'd prefer a different style.